### PR TITLE
2436 additional Json FAT Tests

### LINKED ISSUE #2436: Add additional test scenarios for JSON logging FAT
See https://github.ibm.com/was-liberty/WS-CD-Open/issues/10565

### PR DESCRIPTION
**Pull request by Yushan-Lin**
fixes #2436

### CODE REVIEW

File: dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    public static final String SERVER_XML_TRACE_CONSOLE_ACCESSTRACE_MESSAGE = "traceConsoleAccessTraceMessage.xml";
+    public static final String SERVER_XML_CLEAR_LOGGING_SOURCES = "clearLoggingSources.xml";
+    public static final String SERVER_XML_BASIC = "basicServer.xml";
+    public static final String SERVER_XML_JSON_SOURCE_MESSAGETRACEACCESS = "jsonSourceMessageTraceAccess.xml";
+    public static final String SERVER_XML_JSON_MESSAGE_ACCESS = "jsonMessageSourceAccessLog.xml";
+
+    public static final String SERVER_XML_JSON_MESSAGES = "jsonMessagesLogServer.xml";
+    public static final String SERVER_XML_JSON_CONSOLE = "jsonConsoleLogServer.xml";
+
+    ArrayList<String> sourceList = new ArrayList<String>(Arrays.asList("message", "trace", "accesslog", "ffdc"));
```
Comment (donbourne) [inline dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java]: rename sourceList to ALL_SOURCES_LIST

File: dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+    public static final String SERVER_XML_BASIC_CONSOLE_JSON_MESSAGE_ACCESSTRACE = "basicConsoleJsonMessageTraceAccess.xml";
+    public static final String SERVER_XML_JSON_CONSOLE_JSON_MESSAGE_TRACE_ENABLED = "jsonConsoleJsonMessageTraceEnabled.xml";
+    public static final String SERVER_XML_TRACE_CONSOLE_ACCESSTRACE_MESSAGE = "traceConsoleAccessTraceMessage.xml";
+    public static final String SERVER_XML_CLEAR_LOGGING_SOURCES = "clearLoggingSources.xml";
+    public static final String SERVER_XML_BASIC = "basicServer.xml";
+    public static final String SERVER_XML_JSON_SOURCE_MESSAGETRACEACCESS = "jsonSourceMessageTraceAccess.xml";
+    public static final String SERVER_XML_JSON_MESSAGE_ACCESS = "jsonMessageSourceAccessLog.xml";
+
+    public static final String SERVER_XML_JSON_MESSAGES = "jsonMessagesLogServer.xml";
+    public static final String SERVER_XML_JSON_CONSOLE = "jsonConsoleLogServer.xml";
```
Comment (donbourne) [inline dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java]: remove lines 58 and 59

**Comment by Yushan-Lin** [comment]
#build

File: dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java [hunk 3]
```diff
@@ -78,119 +75,119 @@ public void testError() throws Exception {
...
         RemoteFile consoleLogFile = server.getConsoleLogFile();
-        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", sourceList, "");
+        consoleline = checkConsoleLogUpdate(false, consoleLogFile, "INFO", ALL_SOURCE_LIST, "");
     }
 
     //<logging consoleLogLevel="INFO" consoleFormat="basic" messageFormat="json" messageSource="trace,accesslog" maxFileSize="100"/>
     @Test
     public void testJsonMessageSrcTraceAccess() throws Exception {
-        ArrayList<String> messagesourceList = new ArrayList<String>(Arrays.asList("trace", "accesslog"));
+        ArrayList<String> messageALL_SOURCE_LIST = new ArrayList<String>(Arrays.asList("trace", "accesslog"));
```
Comment (donbourne) [inline dev/com.ibm.ws.logging.json_fat/fat/src/com/ibm/ws/logging/json/fat/JsonConfigTest.java]: in previous review comments, I just meant to change the one static variable to ALL_SOURCE_LIST.  In places where it's not a static variable, should just keep the old names.  eg. messageSourceList here.

**Comment by Yushan-Lin** [comment]
#build

**Pull request merged**